Packages/com.unity.inputsystem/Documentation~/Gamepad.md

```diff
@@ -64,25 +60,76 @@ Increased frequency should lead to an increased number of events on the respecti
 
 ## Rumble
 
-    ////TODO: provide list of what platforms/devices we support rumble on
-
-Gamepads implement the `IDualMotorRumble` interface to control left and right motor speed. Note, however, that not all gamepads support rumble at this point (it is not supported on WebGL, for example).
+The [`Gamepad`](../api/UnityEngine.InputSystem.Gamepad.html) class implement the [`IDualMotorRumble`](../api/UnityEngine.InputSystem.Haptics.IDualMotorRumble.html) interface to allow you to control the left and right motor speeds.
```

Comment:
"implements"

Reply:
Probably useful here to mention left=low-frequency and right=high-frequency motor. Technically, it's up to the gamepad in use which side has the low-freq and which one the high-freq motor but not sure there's actually deviation out there as to which is which. At least PS4 and Xbox match IIRC.